src/table.ts

```diff
@@ -29,13 +29,121 @@ import {Filter} from './filter';
 import {Mutation} from './mutation';
 import {Row} from './row';
 import {ChunkTransformer} from './chunktransformer';
+import {CallOptions} from 'google-gax';
 
 // See protos/google/rpc/code.proto
 // (4=DEADLINE_EXCEEDED, 10=ABORTED, 14=UNAVAILABLE)
 const RETRYABLE_STATUS_CODES = new Set([4, 10, 14]);
 // (1=CANCELLED)
 const IGNORED_STATUS_CODES = new Set([1]);
 
+/**
+ * @typedef {object} Policy
+ * @property {number} [version] Specifies the format of the policy.
+ *     Valid values are 0, 1, and 3. Requests specifying an invalid value will
+ *     be rejected.
+ *
+ *     Operations affecting conditional bindings must specify version 3. This
+ *     can be either setting a conditional policy, modifying a conditional
+ *     binding, or removing a binding (conditional or unconditional) from the
+ *     stored conditional policy.
+ *     Operations on non-conditional policies may specify any valid value or
+ *     leave the field unset.
```

Comment:
@AVaksman where did you find this paragraph, or did you write it? I don't see it in the java pr, but I think it is nice to have.

Reply:
I took it from the [proto](https://github.com/googleapis/googleapis/blob/master/google/iam/v1/policy.proto#L91) files. I thought it explains nicely about the connection between version and conditional binding.